docs/nuxt.config.js

```diff
@@ -20,7 +20,7 @@ renderer.code = (code, language) => {
 renderer.heading = function(text, level, raw, slugger) {
   const getTextMarkup = text => `<span class="bd-content-title">${text}</span>`
 
-  if (!this.options.headerIds || ANCHOR_LINK_HEADING_LEVELS.indexOf(level) === -1) {
+  if (!this.options.headerIds) {
```

Comment:
Since page table of contents, and search require the existence of ID's on the H1, H2 and H3 tags, do we need to do this check?

Reply:
It's the same behavior `marked` uses:
https://github.com/markedjs/marked/blob/1f5b9a19f532e2e1e3e63ae5efd81af75acf572f/lib/marked.js#L963

Reply:
Although we always need to add header IDs....